docs/_docs/templates.md

```diff
@@ -558,19 +686,27 @@ You can also use this tag to create a link in Markdown as follows:
 {% endraw %}
 ```
 
-### Post URL
+Including {% raw %}{{site.baseurl}}{% endraw %} is optional -- it depends on whether you want the link to be absolute or root-relative.
```

Comment:
i was having trouble using the new link and figured i'd come back later to try to understand it more. if you can't use filters with tags, that should probably be noted somewhere, like in the link tag. i'll try to follow up with that, since links are actually a concern of mine.